projects/plugins/jetpack/_inc/client/components/product-activated/index.jsx

```diff
@@ -12,9 +12,15 @@ const ProductActivated = ( { type } ) => {
 		);
 	}
 	return (
-		<div className="jp-product-activated-label">
+		<div
+			className={
+				'jp-product-activated-label' + ( type === 'never-expires' ? ' never-expires' : '' )
+			}
+		>
 			<span className="jp-product-activated-label__text">
-				{ __( 'Subscription active', 'jetpack' ) }
+				{ type === 'never-expires'
+					? __( 'Never expires', 'jetpack' )
+					: __( 'Subscription active', 'jetpack' ) }
```

Comment:
Looks like the build may have tripped up on [this](https://github.com/Automattic/jetpack/actions/runs/4918503296/jobs/8784922187?pr=30519#step:8:8456).

May need to look at: https://www.npmjs.com/package/@automattic/i18n-check-webpack-plugin#user-content-conditional-function-call-compaction

Reply:
Fixed, thanks!